### DISCUSSION TITLE
https://angular.love/angular-for-everyone-how-to-adapt-applications-for-people-with-disabilities

### DISCUSSION THREAD

**Discussion by giscus** [HouseOfAngular/angular-love#314] [General]
# https://angular.love/angular-for-everyone-how-to-adapt-applications-for-people-with-disabilities

Angular.love - a place for all Angular enthusiasts created to inspire and educate.

https://angular.love/angular-for-everyone-how-to-adapt-applications-for-people-with-disabilities

**Comment by hohphu8** [comment]
Good !👍👍
